pkg/client/apiutil/dynamicrestmapper.go

```diff
@@ -143,7 +145,7 @@ func (drm *dynamicRESTMapper) reload() error {
 		return err
 	}
 	if err := drm.setStaticMapper(); err != nil {
-		utilruntime.HandleError(err)
+		log.V(1).Info("reload static RESTMapper", "error", err)
```

Comment:
log.Error here instead, please

Reply:
actually, is there a reason we're not returning the error here?

Reply:
(needs a comment if there is)

Reply:
I don't see a reason not to return the error from `setStaticMapper()` in `reload()`. Errors from that function are those the user should be aware of. I was just keeping the previous behavior of logging.

I'll add a note to audit return statements for go 1.13.

Reply:
yeah, it's fine to bubble up that error, I think.